ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -443,6 +454,31 @@ - (RTCVideoTrack *)createScreenCaptureVideoTrack {
     }
 }
 
+RCT_EXPORT_METHOD(mediaStreamTrackSetVideoEffects:(nonnull NSString *)trackID names:(nonnull NSArray<NSString *> *)names)
+{
+  RTCMediaStreamTrack *track = self.localTracks[trackID];
+  if (track) {
```

Comment:
Better reverse the check and return early if there is no local track.

Reply:
Will do